components/primitives/ProgressBar/progress-bar.tsx

```diff
@@ -0,0 +1,27 @@
+import clsx from "clsx";
+import { FC, HTMLProps } from "react";
+
+interface ProgressBarProps {
+  value: string; // Provide the progress value as a number (0-100)
+  props?: HTMLProps<HTMLProgressElement>;
+  className?: string;
+}
+
+const ProgressBar: FC<ProgressBarProps> = ({ value, className, ...props }) => {
+  return (
+    <progress
+      className={clsx(
```

Comment:
We now have the cn until available👀

So you can try merging the existing PR handling that, and see if you can incorporate it here instead of raw clsx🤝